sdformat_mjcf/sdformat_mjcf/converters/material.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module to convert SDFormat MAterials to MJCF"""
+
+from ignition.math import clamp
+
+from sdformat import Pbr, PbrWorkflow, Material  # noqa: F401
+
+import os
+
+TEXTURE_NUMBER = 0
+
+
+def add_material(geom, material):
+    """
+    Converts an SDFormat material to an MJCF material.
+
+    :param mjcf.Element geom: The MJCF geom to add the material
+    :param sdf.Material material: The SDF material to convert
+    :return: The newly created MJCF material.
+    :rtype: mjcf.Element
+    """
+    pbr = material.pbr_material()
+    sp_color = material.specular()
+    em_color = material.emissive()
+    specular = (sp_color.r() + sp_color.g() + sp_color.b()) / 3.0
+    emissive = (em_color.r() + em_color.g() + em_color.b()) / 3.0
```

Comment:
Taking the average only works if all the rgb values are the same. If, for example, you have a specular of `[0.9, 0, 0]`, I would expect a red highlight, but this would result in gray. I don't know of a solution, but perhaps we can create an issue to remind us to come back to it.